File: magma-platform/magma-platform-bungee/src/main/java/de/jalumu/magma/platform/bungee/text/BungeeTextProvider.java
package de.jalumu.magma.platform.bungee.text;

import de.jalumu.magma.text.Text;
import de.jalumu.magma.text.TextProvider;

public class BungeeTextProvider extends TextProvider {
    @Override
    protected Text text(String text) {
        return new BungeeText().setText(text);
    }
}

File: magma-platform/magma-platform-bungee/src/main/kotlin/de/jalumu/magma/platform/bungee/text/BungeeTextProvider.kt
package de.jalumu.magma.platform.bungee.text

import de.jalumu.magma.text.Text
import de.jalumu.magma.text.TextProvider

class BungeeTextProvider : TextProvider() {
    override fun text(text: String): Text {
        return BungeeText().setText(text)
    }
}
